### ISSUE TITLE
BambuStudio connect multiple accounts to one printer

### ISSUE THREAD

**Issue by hutje** [bambulab/BambuStudio#3533]
Being able to add other users to access the printers from within BambuStudio.
In an educational environment such as a school or library it is needed to give multiple users access to the printers. With the ability to give other users some privileges to send their 3D-print to the printer or seeing the live footage.

The owner can give different users different privileges, for example: controlling the printer, sending sliced objects (with or without remote starting a print privilege) or viewing the livestream.

**Comment by Giladkapel** [comment]
I would also like to see this feature implemented.

**Comment by Jason-Mefford** [comment]
Yes bambu this would be very nice to have!

**Comment by Habasch** [comment]
Please finally implement this function. Creality has done it too. And there are so many people waiting for it.

**Comment by luckygreen** [comment]
Multiuser functionality on the X1C is **way overdue**. When will this be implemented?

**Comment by LarryHiller** [comment]
I bought two x-1 carbons for my classroom.  I dont want to have to make a single account that everyone shares.  Please allow everyone to sign into a machine with their own account.

**Comment by Excds** [comment]
It would be very practical to have this implemented.

**Comment by MartinPavlicek** [comment]
This would be a very useful feature in many scenarios.

**Comment by TheSlider2** [comment]
+1
We have multiple printers at work and I have mine at home. It's a pain to switch accounts each time I need to access one. 
This feature probably also requires the firmware to list multiple authorized accounts in order to also be able to "forget" them.

**Comment by Radot1** [comment]
+1

**Comment by napalmz** [comment]
+1
Also, multi account login on Bambu Studio (or retain login credentials to fast switch between accounts managing printers).
My use case is: Office Print with corporate account and personal printer with personal account.

**Comment by highnze** [comment]
+1

**Comment by dahnutz** [comment]
I subscribe, definitively need multi user support, or share with other account/s or at least dual support for cloud and lan access.

**Comment by NoahK216** [comment]
+1
Please make this happen!

**Comment by AeroDynamicDuck** [comment]
+1 to this! Entirely agree. It's silly to have to unbind the printer between accounts whenever switching.
Or share an account, when the other person doesn't want what I'm printing to affect their feed/history.

**Comment by apiominer** [comment]
Agreed, I'd like to let the rest of my fam use the A1 mini to cloud print.  I'd definitely go through more filament that way--just saying! 😉

**Comment by a-well** [comment]
+1 Please, pretty please

**Comment by GuyH77** [comment]
Please add this

**Comment by lexiko80-LPA** [comment]
+1

**Comment by PIZZA-ALERT** [comment]
ADD IT ALREADY BAMBU FFS

**Comment by daniel-maiorano** [comment]
+1

**Comment by willer** [comment]
Pretty ridiculous that this mistake isn't fixed yet. This was called out two years ago! Do none of you have families, is that it?

**Comment by Giladkapel** [comment]
Creality had the same issue and they added additional 2 users. One is master and there can be 2 more "secondary" users. While this isn't a full solution, I would say it answers for 90% of the multi-user cases.

**Comment by jdagdelen** [comment]
Would like this as well.

**Comment by fishheadcode** [comment]
Same here, my wife uses it in the week, i will use it on the weekends. but i will have my own account, likes, collections and so on. Would be awesome

**Comment by Stefan-Blazek** [comment]
+1

**Comment by BoostMaker** [comment]
What's the status on a solution for connecting multiple user accounts to one printer?

This is not a "nice to have" feature, it is a must have.

**Comment by TraXie666** [comment]
We really need this feature!

**Comment by Rupp3r** [comment]
@DanBao-Bambu any chance that you have some info on the status this feature or could have someone look at this?

Pretty old case and a VERY popular demand. Think the only reason it have not had more interaction before now, is because we all expected it to happen soon and did not want to spam. Same with likes - it is such a common feature that I think most people expected it to be done whether liking the suggestion or not(and it is still the 6'th most liked case and 5'th liked feature suggestion..)

**Comment by DanBao-Bambu** [comment]
Using the LAN mode access of the printer should be able to solve the problem of multiple people sharing the printer, provided that it is under a unified LAN.
@walterwongbbl Please check this requirement.

**Comment by luckygreen** [comment]
@walterwongbbl wrote:
> Using the LAN mode access of the printer should be able to solve the problem of multiple people sharing the printer, provided that it is under a unified LAN.

Neither I nor anyone else that wants this feature, who have bought into the Bambu ecosystem, has any interest in having `to` use LAN mode to share a printer within a family or students in a classroom. (Note that there are obvious follow-up requirements, in particular for classroom use and use within a family. You need at least two roles: regular user (student/child) and an approver role (teacher or parent) with the ability to approve sending an object to print and to perform queue management. I.e. "print _that_ object next, even though it is 12th in queue".

**Comment by sc0ttdav3y** [comment]
Can't believe this isn't a feature. +1

**Comment by MTA-Barry** [comment]
> Using the LAN mode access of the printer should be able to solve the problem of multiple people sharing the printer, provided that it is under a unified LAN. [@walterwongbbl](https://github.com/walterwongbbl) Please check this requirement.

No it would not.
my printer is in a different company LAN than my users. Sometimes they arent even on site and want to print. thats now not possible.

im coming from the Ultimaker ECO-system, wich works flawless.  i can invite users to the printer and make them admin/user, they dont need to add it with a ip adress. and i have a decent print que wich can priotitize prints.

Please please please implement multi user cloud mode.

**Comment by dracon80** [comment]
LAN Only mode isn't really a viable solution as it cripples all the important remote features.

If LAN mode could be enabled and still have the primary account linked, it MIGHT be a solution. But realistically, multi user is an expected and required feature. The fact this is missing just looks bad.

**Comment by apiominer** [comment]
In large part because of the lack of multi-user capability in Bambu Handy, I bought a Prusa CORE One.  My kid was able to print from her phone within minutes of me adding her to my "team" and sending an invite to the printer.  I think Bambu Lab is missing out on a huge aspect of the hobby: parents who want their older kids to be able to participate with them (as well as classroom use).

**Comment by bencardinal** [comment]
100% what @apiominer said!  I recently added a second Bambu printer and would love to be able to share with my child and help teach how to use it without completely changing the account or sharing the account.

**Comment by kusmi** [comment]
Yep - I currently share the account with my child and while it works ok, I now have 100 likes and collected item in my account from my child and can’t find my saved models from makerworld 😉

**Comment by jc5526** [comment]
+1

**Comment by nathanaelmoyer** [comment]
Is there any update on this? I can't believe this still hasn't been addressed.

**Comment by BoostMaker** [comment]
@nathanaelmoyer You can try bugging @DanBao-Bambu about it. They suggested a workaround using LAN mode, but that's not a reasonable option.

Unfortunately, this doesn't seem like an issue for enough people to push Bambu to care about it. I bailed and went to a Prusa because of this.

**Comment by nathanaelmoyer** [comment]
> @nathanaelmoyer You can try bugging @DanBao-Bambu about it. They suggested a workaround using LAN mode, but that's not a reasonable option.
> 
> Unfortunately, this doesn't seem like an issue for enough people to push Bambu to care about it. I bailed and went to a Prusa because of this.

What I don't understand is why I can't at least use LAN mode without disabling the other features. Imagine if regular 2D printers did this...

**Comment by Langelus** [comment]
Definately want this as well.

**Comment by tscreighton** [comment]
Hoping this feature will arrive soon.

**Comment by timothyzalusky** [comment]
@DanBao-Bambu please add this as a high priority feature add, every single company that has purchased a Bambu Printer, specifically E series (X1E) wants this, but maybe has no means to request it, or doesn't know where to ask.  Maybe send out a survey to Bambu Studio users to get an understanding of just what percentage of us really wants this feature.  Please consider.

**Comment by Rupp3r** [comment]
@walterwongbbl & @DanBao-Bambu 
Would you be willing to revisit this issue and give it another consideration? 🙏🏻
Several - good - arguments have been made since your last response about why LAN mode does not actually solve the underlying problem 😅

Thanks!

**Comment by alexdeg92** [comment]
Yup I agree, just bought the h2c and I was really bump when saw that they don't have family account principle it's really not that hard to implement this is a 1-2 week job with one engineer, 1 designer on figma (5-6) hours and review with PM com on
